test/powershell/Modules/Microsoft.PowerShell.Utility/NewTemporaryFile.Tests.ps1

```diff
@@ -14,16 +14,15 @@
         A FileInfo object for the temporary file is returned.
 #>
 
-Describe "NewTemporaryFile" -Tags "CI" {
+Describe "New-TemporaryFile" -Tags "CI" {
 
     It "creates a new temporary file" {
         $tempFile = New-TemporaryFile
 
-        Test-Path $tempFile | Should be $true
+        $tempFile | Should Exist
```

Comment:
From a black box testing point of view, it seems that we should check that the $tempFile path starts with the path returned from GetTempPath()?

Reply:
You could... Would you be ok with this one:
```` powershell
$tempFile | Should BeLikeExactly "$([System.IO.Path]::GetTempPath())*"
````